File: day1/python/solution.py
with open('./input.txt', 'r') as handle:
    text = handle.read().splitlines()
    i = 0
    length = len(text)
    increased = 1
    while i < length:
        if text[i] > text[i-1]:
            increased += 1
        i += 1
    print(increased)

File: day1/go/solution.go
package main

import (
	"bufio"
	"fmt"
	"log"
	"os"
)

func main() {
	output := make([]string, 9999)
	increased := 0
	file, err := os.Open("input.txt")
	if err != nil {
		log.Fatal(err)
	}
	defer func() {
		if err = file.Close(); err != nil {
			log.Fatal(err)
		}
	}()

	scanner := bufio.NewScanner(file)

	for scanner.Scan() {
		output = append(output, scanner.Text())
	}
	for i := 1; i < len(output); i++ {
		if output[i] > output[i-1] {
			increased += 1
		}
	}
	fmt.Println(increased)
}
